### PR TITLE
Added bulk loading api for public keys.

### PR DESCRIPTION
**Pull request by geekbeast**

### CODE REVIEW

File: src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java [hunk 1]
```diff
@@ -43,6 +43,16 @@ private PARAM() {}
     }
 
     /**
+     * @param userIds The id of the user whose public key shall be retrieved.
+     * @return Specified user's public key
+     * @throws ResourceNotFoundException
+     */
+    @Timed
+    @POST( CONTROLLER + PUBLIC_KEY )
+    Map<UUID, PublicKeyEnvelope> getPublicKey( @Path( Names.USERS_FIELD ) Set<UUID> userIds )
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: rename to `getPublicKeys()`, for plural

Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: also, it isn't clear whether or not the keys that are returned are RSA public keys...

Comment (geekbeast) [inline src/main/java/com/kryptnostic/directory/v1/http/DirectoryApi.java]: We don't support any other type of public keys at the moment.  There's some work that needs to be done to also store the public key type, but that's a separate work item.

**Comment by UnsungHero97** [comment]
looks good to me

**Pull request merged**